GitCommands/RevisionReader.cs

```diff
@@ -33,26 +33,30 @@ public sealed class RevisionReader
 
         private readonly GitModule _module;
         private readonly Encoding _logOutputEncoding;
-        private readonly long _sixMonths;
+        private readonly long _oldestBody;
+        private const int _offsetDaysForOldestBody = 6 * 30; // env 6 months
```

Comment:
nit: Did you mean
```suggestion
        private const int _offsetDaysForOldestBody = 6 * 30; // about 6 months
```
?